cypress/e2e/GUI/pagination_and_elements_pag5.cy.js

```diff
@@ -0,0 +1,36 @@
+/// <reference types="Cypress" />
+describe('Paginação', () => {
+    beforeEach(() => {
+      for (let page = 1; page <= 5; page++) {
```

Comment:
Esse `intercept` com um `for` loop no `beforeEach` não faz sentido.

Já falei sobre isso em outro comentário. O mesmo vale aqui.